core/src/main/java/hudson/model/HealthReport.java

```diff
@@ -84,7 +84,7 @@
     /**
      * The path to the icon corresponding to this health score or <code>null</code> to use the default icon
      * corresponding to the current health score.
-     * <p/>
+     * <p>
```

Comment:
I suspect this comes from the stupid default behaviour IntelliJ has about this :-(